Patch obtained from gentoo-overlay

Header licenses and copyright notices:

For Source/WebCore/platform/graphics/GLContext.cpp:
/*
 * Copyright (C) 2011, 2012 Igalia, S.L.
 *
 *  This library is free software; you can redistribute it and/or
 *  modify it under the terms of the GNU Lesser General Public
 *  License as published by the Free Software Foundation; either
 *  version 2 of the License, or (at your option) any later version.
 *
 *  This library is distributed in the hope that it will be useful,
 *  but WITHOUT ANY WARRANTY; without even the implied warranty of
 *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 *  Lesser General Public License for more details.
 *
 *  You should have received a copy of the GNU Lesser General Public
 *  License along with this library; if not, write to the Free Software
 *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
 */

For Source/WebKit/UIProcess/API/glib/WebKitProtocolHandler.cpp:
/*
 * Copyright (C) 2019 Igalia S.L.
 *
 * This library is free software; you can redistribute it and/or
 * modify it under the terms of the GNU Lesser General Public
 * License as published by the Free Software Foundation; either
 * version 2,1 of the License, or (at your option) any later version.
 *
 * This library is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 * Library General Public License for more details.
 *
 * You should have received a copy of the GNU Library General Public License
 * along with this library; see the file COPYING.LIB.  If not, write to
 * the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
 * Boston, MA 02110-1301, USA.
 */

diff -urp webkitgtk-2.33.1.orig/Source/WebCore/platform/graphics/GLContext.cpp webkitgtk-2.33.1/Source/WebCore/platform/graphics/GLContext.cpp
--- webkitgtk-2.33.1.orig/Source/WebCore/platform/graphics/GLContext.cpp	2021-05-13 00:58:57.000000000 -0700
+++ webkitgtk-2.33.1/Source/WebCore/platform/graphics/GLContext.cpp	2021-05-23 22:41:57.610800891 -0700
@@ -31,6 +31,9 @@
 
 #if USE(GLX)
 #include "GLContextGLX.h"
+#endif
+
+#if !USE(OPENGL_ES) && !USE(LIBEPOXY) && !USE(ANGLE)
 #include "OpenGLShims.h"
 #endif
 
Only in webkitgtk-2.33.1/Source/WebCore/platform/graphics: GLContext.cpp.orig
Only in webkitgtk-2.33.1/Source/WebCore/platform/graphics: GLContext.cpp.rej
diff -urp webkitgtk-2.33.1.orig/Source/WebKit/UIProcess/API/glib/WebKitProtocolHandler.cpp webkitgtk-2.33.1/Source/WebKit/UIProcess/API/glib/WebKitProtocolHandler.cpp
--- webkitgtk-2.33.1.orig/Source/WebKit/UIProcess/API/glib/WebKitProtocolHandler.cpp	2021-04-12 01:34:56.000000000 -0700
+++ webkitgtk-2.33.1/Source/WebKit/UIProcess/API/glib/WebKitProtocolHandler.cpp	2021-05-23 22:41:57.620801285 -0700
@@ -65,7 +65,10 @@
 
 #if USE(GLX)
 #include <GL/glx.h>
-#include <WebCore/OpenGLShims.h>
+#endif
+
+#if !USE(OPENGL_ES) && !USE(LIBEPOXY) && !USE(ANGLE)
+#include "WebCore/OpenGLShims.h"
 #endif
 
 #if USE(GSTREAMER)
Only in webkitgtk-2.33.1/Source/WebKit/UIProcess/API/glib: WebKitProtocolHandler.cpp.orig
